### PR TITLE
Shift airtable emailjs gmnhservice

### PR DESCRIPTION
**Pull request by patenish**
involves shift of airtable/emailjs code to using gmnh BE

specifically

1) adding of new user to airtable to save email address
2) query whether a user has already been added in airtable
3) query email address from airtable

### CODE REVIEW

File: src/components/IWantUrNFTForm.vue [hunk 1]
```diff
@@ -16,8 +16,8 @@
         </button>
       </form>
       <form v-if="!isQuestion && !isLoading" @submit.prevent="createAnswer" class="flex-grow">
-        <div v-if="!fromQuestionDetail"><textarea focus-visible type="text" id="nftName" placeholder="Add answer.." class="nes-input gmnh-answer" v-model="nftName" /></div>
-        <div v-else><textarea focus-visible type="text" id="nftName" placeholder="Add answer.." class="nes-input gmnh-answer-detail" v-model="nftName" /></div>
+        <div v-if="!fromQuestionDetail"><textarea focus-visible type="text" id="nftName" placeholder="Write answer here:" class="nes-input gmnh-answer" v-model="nftName" /></div>
```
Comment (patenish) [inline src/components/IWantUrNFTForm.vue]: need to revert this - must be from merge

File: src/components/IWantUrNFTForm.vue [hunk 2]
```diff
@@ -76,7 +76,7 @@ import { PublicKey, Keypair } from '@solana/web3.js';
 import { NodeWallet } from '@metaplex/js';
 import {DEFAULTS} from '@/globals'
 import usePinata from '@/composables/pinata';
-import getWallet from '@/composables/wallet';
+import useWallet from '@/composables/wallet';
```
Comment (patenish) [inline src/components/IWantUrNFTForm.vue]: also revert this

File: src/components/IWantUrNFTForm.vue [hunk 3]
```diff
@@ -201,12 +214,16 @@ export default defineComponent({
             let questionUserIDWallet = undefined
             let ticketLink = formatTicketDetailLink(props.questionID, DEFAULTS.APP_URL)
 
-            retrieveMintFromGMNH(props.questionID) 
+            retrieveByMintId(props.questionID)
```
Comment (patenish) [inline src/components/IWantUrNFTForm.vue]: also fix this

File: src/components/IWantUrNFTForm.vue [hunk 4]
```diff
@@ -285,7 +306,6 @@ export default defineComponent({
   width: 250px;
   height: 250px;
   margin: 16px;
```
Comment (patenish) [inline src/components/IWantUrNFTForm.vue]: add these word-wraps back

File: src/composables/emailNotifications.ts [hunk 5]
```diff
@@ -56,6 +56,17 @@ export function sendEmail(userEmailAddress: string, emailType: string, questionL
   }
 
 
+  console.log(templateParams)
```
Comment (patenish) [inline src/composables/emailNotifications.ts]: remove console log

File: src/composables/emailNotifications.ts [hunk 6]
```diff
@@ -56,6 +56,17 @@ export function sendEmail(userEmailAddress: string, emailType: string, questionL
...
+
```
Comment (patenish) [inline src/composables/emailNotifications.ts]: remove since emailjs worked vs. axios

File: src/composables/gmnh-service.ts [hunk 7]
```diff
@@ -93,9 +93,10 @@ async function retrieveMint(mintId: string): Promise<PNFT[]> {
     data.append('description', description);
     data.append('userWalletAddr', walletAddr.toBase58());
 
+    console.log("url is ", createQuestionUrl)
```
Comment (patenish) [inline src/composables/gmnh-service.ts]: remove console log

**Review by patenish — COMMENTED** [review]
minor updates needed based on merge & cleanup

File: src/components/IWantUrNFTForm.vue [hunk 8]
```diff
@@ -154,6 +153,19 @@ export default defineComponent({
     const { isConnected, getWalletAddress } = getWallet();
     const { clearError, setError } = useError();
 
+    //This is the HelpDesk treasury wallet (9px36ZsECEdSbNAobezC77Wr9BfACenRN1W8X7AUuWAb) where all NFTs will be minted to
```
Comment (karnie6) [inline src/components/IWantUrNFTForm.vue]: why did we add this back?  Can we remove?

File: src/composables/gmnh-service.ts [hunk 9]
```diff
@@ -118,3 +118,45 @@ async function retrieveMint(mintId: string): Promise<PNFT[]> {
...
+    /* Input: user wallet address & user email address
+       Output: boolean of whether user has been successfully added to Airtable
+    */
+    const addUserInfoAirtableUrl = DEFAULTS.GMNH_SERVICE_APP_URL + 'addUserInfoAirtable/';
+  
+    const data = new FormData();
+    data.append('userWalletAddress', userWalletAddress);
+    data.append('userEmailAddress', userEmailAddress);
+
+    console.log("data is ", data)
```
Comment (karnie6) [inline src/composables/gmnh-service.ts]: can we remove this?

**Pull request merged**